src/patterns/data/default/vein_and_venule_pattern.tsv

```diff
@@ -37,3 +37,13 @@ UBERON:8600105	lumbar vein	FMA:70889	UBERON:0001638	UBERON:0002355	A vein that i
 UBERON:8420003	posterior ethmoidal vein	FMA:51772	UBERON:8420001		An ethmoidal vein that accompanies a posterior ethmoidal artery. The posterior ethmoidal vein often drains into a venous network under the orbital roof.	https://doi.org/10.1167/iovs.02-0865|https://doi.org/10.1007/978-3-031-40697-3_23							https://orcid.org/0000-0001-8314-2140	2024-08-28T09:55:00Z
+UBERON:8920010	left marginal vein	FMA:4708	UBERON:0001638	UBERON:0000948	A vein that is a tributary of the great cardiac vein (left coronary vein) and drains blood from the left ventricle of the heart. It courses with the left marginal artery as it travels superiorly along the left cardiac border.	wikipedia:Left_marginal_vein|https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/left-marginal-vein-of-heart/22447				UBERON:0006958		UBERON:0002084	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920011	stellate vein of kidney	FMA:71635	UBERON:0001638	UBERON:0002113	A vein that drains blood from the most superficial zones of the renal cortex. Groups of stellate veins are arranged in a star-like pattern, converging centrally to drain into an interlobular vein.	wikipedia:Stellate_veins|https://www.elsevier.com/resources/anatomy/kidney-lobe/intrarenal-blood-vessels/interlobular-veins-of-kidney/15668	Set of stellate veins of kidney	FMA:71635		UBERON:0005168		UBERON:0001225	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920012	segmental renal vein		UBERON:0001638	UBERON:0002113	A vein that is a tributary of the renal vein and drains blood from a segment of the kidney.	https://www.ncbi.nlm.nih.gov/books/NBK538298/|doi:10.37532/1308-4038.18.11.26				UBERON:0001140			https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920013	anterior ciliary vein	FMA:51798	UBERON:0001638	UBERON:0000970	A vein that is a tributary of the superior opthalmic vein and drains blood from the ciliary body of the eye.	https://www.kenhub.com/en/library/anatomy/ophthalmic-veins|https://www.imaios.com/en/e-Anatomy/Anatomical-Parts/Anterior-ciliary-veins|https://www.nature.com/articles/s41433-023-02416-z				UBERON:0011192		UBERON:0001775	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920014	left bronchial vein	FMA:4837	UBERON:0001592	UBERON:0002048	A bronchial vein that drains blood from the bronchi and hilar structures into the accessory hemiazygos vein or the left superior intercostal vein.	https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/bronchial-veins-right/21896|wikipedia:Bronchial_veins				UBERON:0006665|UBERON:0012195		UBERON:0002185|UBERON:0004886	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33

```

Comment:
I would drop the elsevier anatomy link. The wikipedia and FMA links should suffice.